Web/Edubase.Services.Texuna/ChangeHistory/ChangeHistoryService.cs

```diff
@@ -28,13 +28,13 @@ public async Task<List<UserGroupModel>> GetSuggesterGroupsAsync(IPrincipal princ
         [Obsolete("Please use GetSuggesterGroupsAsync", true)]
         public async Task<List<UserGroupModel>> GetApproversGroupsAsync(IPrincipal principal)
         {
-            throw new NotImplementedException("GetApproversGroupsAsync is not implemented because it returns the exact same information as GetSuggesterGroupsAsync");
+            return await GetSuggesterGroupsAsync(principal);
         }
```

Comment:
Possibly outside the scope of this but if we're modifying this anyway could we just inline this method? Or remove it? 

It appears to have been marked as obsolete for a while and given that it previously threw an exception this suggests it's not used anywhere, so perhaps deletion is the better option.

Reply:
I will check to see how this is used - removing it could be a better move.

Reply:
It was kept due to it being part of the interface specification - the unused part of the interface has been removed too